kanidm_book/src/security_hardening.md

```diff
@@ -121,26 +129,28 @@ This file should be everyone-readable which is why the bits are defined as such.
 The commands provided in this book will run kanidmd as "root" in the container to make the onboarding
 smoother. However, this is not recommended in production for security reasons.
 
-You should allocate a uidnumber/gidnumber for the service to run as that is unique on your host
+You should allocate unique uid and gid numbers for the service to run as on your host
 system. In this example we use `1000:1000`
 
 You will need to adjust the permissions on the `/data` volume to ensure that the process
 can manage the files. Kanidm requires the ability to write to the `/data` directory to create
-the sqlite files. This uid/gidnumber should match the above. You could consider the following
+the sqlite files. This uid/gid number should match the above. You could consider the following
 changes to help isolate these changes:
 
     docker run --rm -i -t -v kanidmd:/data opensuse/leap:latest /bin/sh
-    # mkdir /data/db/
-    # chown 1000:1000 /data/db/
-    # chmod 750 /data/db/
-    # sed -i -e "s/db_path.*/db_path = \"\/data\/db\/kanidm.db\"/g" /data/server.toml
-    # chown root:root /data/server.toml
-    # chmod 644 /data/server.toml
+    mkdir /data/db/
+    chown 1000:1000 /data/db/
+    chmod 750 /data/db/
+    sed -i -e "s/db_path.*/db_path = \"\/data\/db\/kanidm.db\"/g" /data/server.toml
+    chown root:root /data/server.toml
+    chmod 644 /data/server.toml
+    
+    <!-- removed prompts for consistency with other command examples -->
```

Comment:
I feel like this needs a note to say the commands after `docker run...` need to run in the container. Is there an easy/coherent way to note this?

Reply:
I am not docker-fluent, do the commands need to change? Or are the commands correct as-is, and is it enough to say something like "these commands must be in the container"

Reply:
The latter - the commands are correct and the note would make sense as you have it there - apologies for being unclear 😄